src/services/polkadotExtension.ts

```diff
@@ -236,7 +236,11 @@ export const getHydraDxAccountsFromExtension: () => Promise<
       account.meta.genesisHash &&
       (extStore.genesisHash === account.meta.genesisHash ||
         account.meta.genesisHash ===
-        '0x0ed32bfcab4a83517fac88f2aa7cbc2f88d3ab93be9a12b6188a036bf8a943c2')
+        '0x0ed32bfcab4a83517fac88f2aa7cbc2f88d3ab93be9a12b6188a036bf8a943c2' ||
```

Comment:
https://github.com/jak-pan/common/blob/78ebb6b74b5c2c244ebbf2dc5207dc117530133e/packages/networks/src/defaults.ts#L18 these are the hashes we need.

Reply:
sorry, mistaken the gen2 with one of the gen3. now both gen3.0 and gen3.1 are in (and the previously included snakenet genesis hash)